src/__tests__/CqlLibrary.test.ts

```diff
@@ -0,0 +1,25 @@
+import { CqlLibrary } from "../CqlLibrary";
+import { Model } from "../Model";
+
+test("Verifies that Measure has the correct attributes", () => {
```

Comment:
Test description needs to be library specific

Reply:
Yeah I have just seen that, thank you I have changed it